ffc-pay-alerts/schema/event-schema.js

```diff
@@ -0,0 +1,125 @@
+module.exports = [{
+  eventType: 'siti-inbound',
+  service: 'siti inbound',
+  notifyTemplateId: '48963d01-eb09-439c-a12e-e99129127b06',
+  externalProcess: true,

```

Comment:
Are these needed, are we using it?

Reply:
I think we do, all the event listed are those being accessed by` ffc-pay/web` for reference, see `ffc-pay-web\app\routes\models\event-projection-schema.js`

Reply:
Hmmmm that seems to be used as metadata to help articulate the graph better -- we dnt have that functionality need in this repo, just a mapping between event and template